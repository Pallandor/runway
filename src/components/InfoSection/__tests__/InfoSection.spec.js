import React from 'react';
import { shallow } from 'enzyme';
import InfoSection from '..';

describe('InfoSection', () => {
  it('renders correctly with props', () => {
    const requiredProps = {
      content: 'mock content',
      height: '100px',
      width: '200px'
    };
    const component = shallow(<InfoSection {...requiredProps} />);
    expect(component).toMatchSnapshot();
  });
});
