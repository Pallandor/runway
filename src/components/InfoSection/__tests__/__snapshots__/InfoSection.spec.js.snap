// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoSection renders correctly with props 1`] = `
.emotion-2 {
  height: 100px;
  width: 200px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  margin-right: 10px;
}

.emotion-0 {
  fill: #323232;
  -webkit-flex: 0 0 21px;
  -ms-flex: 0 0 21px;
  flex: 0 0 21px;
  margin-right: 11px;
  position: relative;
  top: 3px;
}

.emotion-1 {
  color: #323232;
  line-height: 1.21;
  font-size: 14px;
  font-family: Ciutadella;
}

<div
  className="emotion-2"
>
  <InfoIcon
    className="emotion-0"
    height="21px"
    width="21px"
  />
  <span
    className="emotion-1"
  >
    mock content
  </span>
</div>
`;
